presto-spark-base/src/main/java/com/facebook/presto/spark/execution/property/NativeExecutionSystemConfig.java

```diff
@@ -104,6 +107,9 @@ public class NativeExecutionSystemConfig
     private String httpsCertPath = "";
     private String httpsKeyPath = "";
     private int numIoThreads = 30;
+    private boolean asyncDataCacheEnabled;
```

Comment:
Should these have default values? true, 0 and 30 (according to documentation).

Reply:
We set it to false here to disable it.  The default values are for when it is not overwritten here.

Reply:
Would you explicitly set it or make a comment that default is 'false'? 

```
private boolean asyncDataCacheEnabled = false;
```

Reply:
> The default values are for when it is not overwritten here.

I'm not sure I understand.... The user reading the documentation would assume that default is what they get without changing anything. It looks like actual default is false, no?

Reply:
Setting it explicitly violate a lint rule.  I will put a comment.

The default is you don't overwrite it in application code.  Here POS is overwriting it explicitly so the defaults no longer applies.  Do you have a place I can write POS defaults?